test/SlowTests/Server/Replication/ReplicationCleanTombstones.cs

```diff
@@ -879,5 +862,53 @@ public async Task CanBackupAndRestoreTombstonesWithSameId(Options options)
                 }
             }
         }
+
+        private async Task<long> CreateAndRunBackup(IDocumentStore store, RavenDatabaseMode mode, string backupPath)
+        {
+            var config = Backup.CreateBackupConfiguration(backupPath);
+
+            if (mode == RavenDatabaseMode.Single)
+                return await Backup.UpdateConfigAndRunBackupAsync(Server, config, store);
+            
+            var backupCompleted = await Sharding.Backup.WaitForBackupToComplete(store);
+            var backupTaskId = await Sharding.Backup.UpdateConfigurationAndRunBackupAsync(Server, store, config);
+            Assert.True(WaitHandle.WaitAll(backupCompleted, TimeSpan.FromSeconds(10)));
+
+            return backupTaskId;
+        }
+
+        private async Task<RestoreBackupConfiguration> RunBackupAndCreateRestoreConfiguration(DocumentStore store, RavenDatabaseMode mode, long backupTaskId, string backupPath)
```

Comment:
does it make sense to move this to the backup class?

Reply:
`RunBackupAndCreateRestoreConfigurationAsync`

Reply:
done